README.md

```diff
@@ -12,18 +12,20 @@ A `.env` file containing environment variables necessary for the application's o
 
 Please duplicate the `./client/.env` file into `./client/.env.local` for local development. `.env.local` has been added to the `.gitignore` to avoid unintentional upload of the environment variables.
 
-## Local Docker Swarm Deployment
-Build docker images:
-```shell script
-docker build -t docker.uncharted.software/uncharted-aske/hmi/client -f Dockerfile.client .
-```
+## Docker Compose
+The client's docker image requires configuration via environment files. The environment files that are required
+are `./client/.env.nginx` and `./client/.env.dist`. See Confluence for default configuration parameters.
```

Comment:
Now that the project is public, we might avoid using the term _Confluence_ and use something like `Contact Uncharted...`, same thing for the _Deployment_ instructions.